lib/IRGen/GenDecl.cpp

```diff
@@ -2335,7 +2335,7 @@ llvm::Constant *IRGenModule::emitProtocolConformances() {
     sectionName = "__TEXT, __swift2_proto, regular, no_dead_strip";
     break;
   case llvm::Triple::ELF:
-    sectionName = ".swift2_protocol_conformances";
+    sectionName = "swift2_protocol_conformances";
```

Comment:
Can we keep the existing symbol names to avoid unnecessary ABI thrash? I don't know what other tools might be looking for these symbol names.

Reply:
No, that is part of the magic.  The reason for changing the section names here is to permit the linker to generate the start and stop markers.

Reply:
Can the reflection metadata symbols still be found out-of-process by the remote reflection library?

Reply:
No, that is what Im seeing now.  The protocol conformances and the type metadata works, but the reflection metadata is causing some problems.  The section exists and it can find the section (as it is not coalesced into `.data`).  The problem is that there is a section load offset that needs to be accounted for (as otherwise the relative pointer is now pointing to the wrong location) or `swift-reflection-dump` for example gets things wrong.